GTMAppAuthSwift/Sources/GTMAppAuthFetcherAuthorization.swift

```diff
@@ -39,25 +39,24 @@ import GTMSessionFetcher
   /// Retrieves the saved authorization for the supplied name.
   ///
   /// - Parameter itemName: The `String` name for the save authorization.
-  /// - Throws: `Error.failedToRetrieveAuthorizationFromKeychain` if retrieving the authorization
-  ///   failed.
+  /// - Throws: `Error.failedToConvertKeychainDataToAuthorization` if converting the authorization
```

Comment:
Should we mention `GTMKeychainError` here since we're not catching these?

Reply:
Added here and throughout this file.